### ISSUE TITLE
No changes detected in port overrides when port number isn't changing.

### ISSUE THREAD

**Issue by wolf-cosmose** [paultyng/terraform-provider-unifi#121]
I suspect this is because of the new port override hashing method.

I have for example:
```
  port_override {
    number          = 6
    name            = "office switch"
    port_profile_id = data.unifi_port_profile.builtin["office"].id
  }
```
If I change the port name or profile ID and run `terraform plan`, it says "No changes. Infrastructure is up-to-date."

I need to add another port_override or change some port_override's number for it to detect any changes,
and even then it will say it's deleting the whole port_override section and adding a new one,
instead of updating in-place.

**Comment by paultyng** [comment]
This is probably related to my set logic customization, sorry about that, let me just restore it to the default behavior and see how that goes.

**Comment by paultyng** [comment]
It would be a lot easier once rich maps are supported in the SDK, but not there yet.

**Comment by paultyng** [comment]
Just linking this for future reference: https://github.com/hashicorp/terraform-plugin-sdk/issues/62

**Comment by paultyng** [comment]
Just kicked off the release for 0.24.2. The diff looks weird (a bunch of adds and removes of port overrides, just for like a name change), but its at least working for now.

**Comment by paultyng** [comment]
Fixed in 4b078694174c2de55dfdf9419ea48cc9f14d10b9
